rest-api-spec/api/indices.synced_flush.json

```diff
@@ -0,0 +1,17 @@
+{
+  "indices.syncedflush": {
+    "documentation": "http://www.elastic.co/guide/en/elasticsearch/reference/master/indices-synced-flush.html",
+    "methods": ["POST", "GET"],
+    "url": {
+      "path": "/_syncedflush",
```

Comment:
I am not sure about the name, I think we should call it `_seal` maybe @clintongormley has a better idea